numpyro/infer/initialization.py

```diff
@@ -40,7 +40,13 @@ def init_to_median(site=None, num_samples=15):
             samples = site["fn"](
                 sample_shape=(num_samples,) + sample_shape, rng_key=rng_key
             )
-            return jnp.median(samples, axis=0)
+            from numpyro.infer.util import helpful_support_errors
```

Comment:
Just curious, you put the import here because of circular import?

Reply:
Yes, I borrowed that line from `init_to_uniform` here.

https://github.com/pyro-ppl/numpyro/blob/627d19a8e01e3d1b6c7a594ef7f49face4921f92/numpyro/infer/initialization.py#L108-L109